src/undo_log/undo_log.c

```diff
@@ -373,10 +367,10 @@ void UndoLog_CreateNode
 // undo edge creation
 void UndoLog_CreateEdge
 (
-	UndoLog *log,
+	UndoLog log,           // undo log
 	Edge *edge             // edge created
```

Comment:
Just making sure the comment starts at a distance of 2 spaces from the furthest `,`